packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -95,6 +95,23 @@ pub struct ExperimentConfig {
     clap(global = true, short = 'w', long, default_value_t = num_cpus::get(), validator = at_least_one, env = "HASH_WORKERS")
     )]
     pub num_workers: usize,
+
+    /// Allows a JavaScript runner to grow to some initial size in megabytes before triggering
+    /// garbage collections.
+    ///
+    /// This is useful when it is known that experiments need a certain minimum heap to run to
+    /// avoid repeatedly invoking the garbage collector when growing the heap.
+    /// When used in the wrong conditions this could waste memory.
+    #[cfg_attr(feature = "clap", clap(global = true, long))]
+    pub js_runner_initial_heap_constraint: Option<usize>,
+
+    /// Max size of the JS runner heap in MB.
```

Comment:
Only a nit: At the `initial_heap_constraint`, we write `megabytes`, here it's `MB`. For the sake of consistency, could we change one? Feel free to ignore this, if you don't agree.

Reply:
Changed in https://github.com/hashintel/hash/pull/446/commits/093e7978530a5f503f394212a205e1ed5dc0006a